mainwindow.cpp

```diff
@@ -169,5 +175,47 @@ void MainWindow::mapping_card_value()
     else {
         cardArr &= ~(1UL << 15);
     }
-//    if (cardArr.count)
+
+    //handle move count
+    oldCardArr ^= cardArr;
+    for(; oldCardArr>0;)
+    {
+        if(oldCardArr%2)
+        {
+            numberOfMove++;
+            numberOfOpenCard++;
+        }
+        oldCardArr = oldCardArr >> 1;
```

Comment:
Have you tested the code yet? When I checked 2 cards, the count number is added 2.